itkwidgets/widget_viewer.py

```diff
@@ -624,6 +636,66 @@ def view(image=None,
         widget.
     """
 
+    # this block allows the user to pass already formed vtkActor vtkVolume objects
+    actors = kwargs.pop("actors", None)
+    if have_vtk and actors is not None:
+        if not isinstance(actors, (list, tuple)): # passing the object directly, so make it a list
+            actors = [actors]
+        
+        images = []
+
+        for a in actors:
+
+            if isinstance(a, vtk.vtkAssembly): #unpack assemblies
+                cl = vtk.vtkPropCollection()
+                a.GetActors(cl)
+                cl.InitTraversal()
+                for i in range(a.GetNumberOfPaths()):
+                    ac = vtk.vtkActor.SafeDownCast(cl.GetNextProp())
+                    apoly = ac.GetMapper().GetInput()
+                    prop = ac.GetProperty()
+                    transform = vtk.vtkTransform()
+                    transform.SetMatrix(ac.GetMatrix())
+                    tp = vtk.vtkTransformPolyDataFilter()
+                    tp.SetTransform(transform)
+                    tp.SetInputData(apoly)
+                    tp.Update()
+                    poly = tp.GetOutput()
+                    if poly.GetNumberOfPolys():
+                        geometries.append(poly)
```

Comment:
Since the index in the `geometry_colors` list corresponds to the index in the `geometries` list, we need to use `insert` istead of `append` here, too.